### DISCUSSION TITLE
Strange issue with 3.10.1

### DISCUSSION THREAD

**Discussion by jharris110478** [nextgenhealthcare/connect#5488] [Q&A] [answered]
We have noticed that Mirth is sending messages over and over. It does not make sense considering Mirth shouldn’t have any direct communication with Copia (Orchard LIS) . Mirth should only be communicating with the local folders on the server.

**Comment by pacmano1** [comment]
You are going to need to provide A LOT more detail than that.   

Mirth doesn't send stuff unless you tell it to.    It doesn't retry unless you tell it to.

**Reply by jharris110478** [reply]
Orchard is dropping a file to a folder on the server Mirth picks it up via file reader and sends a JSON via an API call . The JSON file is then converted back to an HL7 with a base64 insertion from the JSON and sent to another folder in which Orchard picks up and inserts into their LIS. For whatever reason when Mirth is running that file comes back into the folder from Orchard and cycles back through.

**Reply by pacmano1** [reply]
1. Orchard sends a file to /somedir
2. Mirth reads from /somedir
3. Mirth sends JSON to some API and gets a response "R"
4. "R" or part of "R" is then used by Mirth to create an HL7 message "A"
5. "A" is written to /somedir2
6. Orchard reads A from /somedir2

Not sure that is correct, but you need to be explicit when you ask for help.

I don't know what your last sentence mean, "that file" and "the folder" are not clear.

**Comment by siddharthsharma9258** [comment, marked as answer]
If you are reading a file, and not telling Mirth to move-after-read, its going to read over and over again.

**Reply by pacmano1** [reply]
Yes, @siddharthsharma9258, an excellent obvious point!

**Reply by jharris110478** [reply]
Mirth was moving and deleting the file. It appears the issue is now resolved. I think Orchard was resending the result because it was not marked F for Final in the OBX 11.1 . I corrected that in the code. Thanks for the input.
